### PR TITLE
PMM-6789 Add connection column

### PR DESCRIPTION
**Pull request by tiagomotasantos**
[PMM-6789](https://jira.percona.com/browse/PMM-6789)

### CODE REVIEW

File: pmm-app/src/pmm-dbaas/components/XtraDB/XtraDB.service.ts [hunk 1]
```diff
@@ -32,14 +44,14 @@ const toAPI = (xtradbCluster: XtraDBCluster): XtraDBClusterPayload => ({
     cluster_size: xtradbCluster.clusterSize,
     pxc: {
       compute_resources: {
-        cpu_m: xtradbCluster.cpu,
-        memory_bytes: xtradbCluster.memory * 1024,
+        cpu_m: xtradbCluster.cpu * 1000,
```
Comment (atymchuk) [inline pmm-app/src/pmm-dbaas/components/XtraDB/XtraDB.service.ts:47]: Shouldn't those figure be multiples of 1024?

**Review by atymchuk — DISMISSED** [review]

File: pmm-app/src/pmm-dbaas/components/XtraDB/XtraDB.service.ts (hunk 1 above)
Comment (tiagomotasantos) [reply pmm-app/src/pmm-dbaas/components/XtraDB/XtraDB.service.ts:47]: I thought so too but we are using [DecimalSI](https://github.com/kubernetes/apimachinery/blob/master/pkg/api/resource/quantity.go#L119) format on the backend. I tested with 2000000000 and I got 2Gb as expected like shown in the following image 
Screenshot 2020-10-27 at 11 00 20

**Review by lunaticusgreen — COMMENTED** [review]
Looks good

**Comment by lunaticusgreen** [comment]
@tiagomotasantos pls update branch

**Review by lunaticusgreen — APPROVED** [review]
LGTM 👍

**Pull request merged**